notebooks/ConjugateMap.ipynb

```diff
@@ -54,7 +54,7 @@
     "\n",
     "# Import functions:\n",
     "import datetime\n",
-    "from conjugate_map import *"
+    "from conjugate_map.conjCalcFunctions import *"
```

Comment:
```suggestion
    "import conjugate_map.conjCalcFunctions as calc_func"
```
 If not now, recommend making an issue for this.

Reply:
Captured in Issue #17 .